lib/rules/no-unsupported-features/node-builtins.js

```diff
@@ -31,6 +31,200 @@ const traceMap = {
                 paths: { [READ]: { supported: ["8.9.0"] } },
             },
         },
+
+        // General APIs
```

Comment:
I wanted to make a custom list from the node globals
https://nodejs.org/api/globals.html (via https://nodejs.org/api/globals.json)

As the numbers are quite different between mdn and the node docs 

If I find the time I will see if I can make a script to scrape that data, but so far I have found the scripts to be lacking 😅